lib/cli/github_settings/do_set_branch_protection.js

```diff
@@ -152,33 +163,86 @@ function do_set_branch_protection(subcmd, opts, args, cb) {
+                            ' to push to "master" ' +
+                            'branch of %s ' +
+                            '(https://github.com/joyent/%s/settings/branches)';
+                        reqOpts = {
+                            owner: 'joyent',
+                            repo: repoName,
+                            branch: 'master',
+
+                            enforce_admins: true,
+                            required_pull_request_reviews: null,
+                            restrictions: {
+                                users: ctx.unlockUsers,
+                                teams: []
+                            },
+                            required_status_checks: null
+                        };
+                        // fml
+                        if (
+                            ctx.unlockUsers &&
+                            ctx.masterProt &&
+                            ctx.masterProt.required_status_checks &&
+                            ctx.masterProt.required_status_checks
+                                .contexts &&
+                            ctx.masterProt.required_status_checks.contexts
+                                .length > 0
+                        ) {
+                            lockMessage = lockMessage + '\n' +
+                                'The following status checks were ' +
+                                'disabled:\n ' +
+                                ctx.masterProt.required_status_checks.contexts.join(' \n');
+                        }
+                    } else {
+                        lockMessage =
+                            'Applying standard branch protection ' +
+                            'rules to "master" branch on repo "%s" ' +
+                            '(https://github.com/joyent/%s/settings/branches)';
+                        reqOpts = {
+                            owner: 'joyent',
+                            repo: repoName,
+                            branch: 'master',
+
+                            enforce_admins: true,
+                            required_pull_request_reviews: {
+                                dismissal_restrictions: {},
+                                dismiss_stale_reviews: true,
+                                require_code_owner_reviews: false,
+                                required_approving_review_count: 1
+                            },
+                            restrictions: null
+                        };
+                        if (ctx.statuses) {
+                            reqOpts.required_status_checks = {
+                                contexts: ctx.statuses,
+                                strict: true
+                            };
+                            lockMessage = lockMessage + '\n' +
+                                'The following status checks ' +
+                                'were enabled:\n  ' +
+                                ctx.statuses.join('\n  ');
```

Comment:
4-space indent in the lockMessage please

Reply:
I don't follow - it is already 4-space indented? Should line 231 be further indented?

Reply:
It's ok, prettier told me what it needed.

Reply:
As above I meant about the printed message. `prettier` can handle the *code* indentation.

```
'were enabled:\n    ' +
ctx.statuses.join('\n    ');
```

Reply:
Got it, thanks.